src/utils/tickutils.go

```diff
@@ -35,7 +35,6 @@ func TickUntilDone(ctx context.Context, refreshRate int64, action func() error)
 		case <-ctx.Done():
 			return ctx.Err() // Stop execution if end signal received
 		case <-ticker.C:
-			break
```

Comment:
I'm not sure I understand this, the `case` has no statements?

Reply:
The break is useless, as it exits from the select. Which will happen even without it. The ticker is just to exit the blocking select call and execute action.

Reply:
leaving it generated a warning, hence the removal.

Reply:
Gotcha. Can you add a comment for each of those cases? It took me a lil while to realise it.

Reply:
Yep, will do!